crates/serde/src/storage.rs

```diff
@@ -26,13 +43,31 @@ use serde::{Deserialize, Deserializer, Serialize};
 ///
 /// The contained [B256] and From implementation for String are used to preserve the input and
 /// implement this behavior from geth.
-#[derive(Clone, Copy, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
-#[serde(from = "U256", into = "String")]
-pub struct JsonStorageKey(pub B256);
+#[derive(Clone, Copy, Debug, Default, PartialEq, Eq, Serialize)]
+#[serde(into = "String")]
+pub struct JsonStorageKey(pub StorageKeyKind);
```

Comment:
this type needs a function to get the value as B256

Reply:
Addressed in [c6e85f9](https://github.com/alloy-rs/alloy/pull/1597/commits/c6e85f97d6df336a761804042f8793971d6fe397)